install.sh

```diff
@@ -308,14 +320,16 @@ compile_tmux(){
 
 tmux_download(){
   echo ":Downloading tmux source code"
-  echo ":N.B: This will take a while. To speed this up install tmux 1.9 manually on your machine and restart this process."
+  echo ":N.B: This will take a while. To speed this up install tmux 2.2 manually on your machine and restart this process."
   
   echo ":Downloading Libevent..."
-  DOWNLOAD https://raw.githubusercontent.com/c9/install/master/packages/tmux/libevent-2.0.21-stable.tar.gz libevent-2.0.21-stable.tar.gz
+  # DOWNLOAD https://raw.githubusercontent.com/c9/install/master/packages/tmux/libevent-2.0.21-stable.tar.gz
+  DOWNLOAD https://github.com/libevent/libevent/releases/download/release-2.0.22-stable/libevent-2.0.22-stable.tar.gz libevent-2.0.22-stable.tar.gz
   echo ":Downloading Ncurses..."
-  DOWNLOAD https://raw.githubusercontent.com/c9/install/master/packages/tmux/ncurses-5.9.tar.gz ncurses-5.9.tar.gz
+  DOWNLOAD https://github.com/c9/install/raw/replace-pty-js/packages/tmux/ncurses-6.0.tar.gz ncurses-6.0.tar.gz
```

Comment:
Consider copying the package binary to the master branch as well, and downloading from there

Reply:
Will update after merging this cr.